### DISCUSSION TITLE
Problem with already established connection

### DISCUSSION THREAD

**Discussion by swissmaster1** [mKeRix/room-assistant#344] [Q&A]
Hi there
I have installed room assistant on a PI Zero on which I used a python script which is listening on mqtt and therefore using the connection to the mqtt broker. For this reason I get an error on starting room assistant because I think connection is already used:
11/13/2020, 9:55:52 AM - error - ClusterService: getaddrinfo -3008

When I stop my python script, room assistant will run smoothly.
Is there something I can do to run both on same raspberry?

**Comment by swissmaster1** [comment]
It looks that I‘m wrong. It must be a diffrent problem. Room Assistant stops after every reboot recognising bluetooth devices. At beginning it works for a random time, but then it stops working. In Syslog i cant find anything. Does room assistant have its own log file somewhere? If yes where?

**Reply by mKeRix** [reply]
The getaddr error you saw is related to the mDNS auto discovery, that shouldn't cause issues in most cases. room-assistant does not do any log management of its own, but if you run it with systemd you can look at `/var/syslog`. If you want a filtered view you can use `journalctl -u room-assistant.service`.
